scripts/linux_arm64_librdkafka_install.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+TEMP=/tmp/librdkafka
+LIB_PATH=/usr/local/lib
```

Comment:
If you use `LIB_PATH=${LIB_PATH:-/usr/local/lib}`, it allows this to be set from outside the script.

This is variable expansion. The `:-` is "If LIB_PATH is empty, expand to the following."
So basically, if someone has `export LIB_PATH='/usr/bin/'` in their environment, and runs this script, that's the path that would get looked at.

If you made that change here, you'd want to make the same change to `scripts/linux-arm64-dependency-check.sh`.